package.json

```diff
@@ -20,7 +20,7 @@
     "remove-dist": "rimraf ./dist",
     "production": "node tools/distServer.js",
     "prebuild": "npm run clean-dist && npm run lint && npm run test",
-    "build": "babel-node tools/build.js && node tools/distServer.js",
+    "build": "babel-node tools/build.js",
```

Comment:
Add the commando to start express:

"build": "babel-node tools/build.js && node tools/distServer.js",